state/txindex/indexer.go

```diff
@@ -7,6 +7,7 @@ import (
 	abci "github.com/cometbft/cometbft/abci/types"
 	"github.com/cometbft/cometbft/internal/pubsub/query"
 	"github.com/cometbft/cometbft/libs/log"
+	ctypes "github.com/cometbft/cometbft/rpc/core/types"
```

Comment:
We use `ctypes` for consensus types in the code. Can we rename this to something recalling rpc?

Reply:
Removed need for it from your suggestion here https://github.com/cometbft/cometbft/pull/2855/commits/ee94683447e945094726f70a0b73790a0b7a7bb9